docs/api.md

```diff
@@ -1198,6 +1201,85 @@ Successful requests will produce a "200 OK" response with the random bytes as he
 There are no standard failure modes for this endpoint.
 
 
+## POST /v1/oauth/sessions/revoke
+
+:lock: Authenticated using an OAuth bearer token, with the `account` scope.
```

Comment:
I think it's the Right Thing to use a different scope here than we used over in https://github.com/mozilla/fxa-oauth-server/pull/262/files, but we should give it a bit of thought.